src/components/header/header-cell.component.ts

```diff
@@ -1,9 +1,7 @@
-import {
-  Component, Input, EventEmitter, Output, HostBinding, HostListener
```

Comment:
@amcdnl This just broke the build. `HostListener` import is needed.

This PR is reverting some of the earlier header context menu code.